assets/src/edit-story/components/panels/design/borderRadius/borderRadius.js

```diff
@@ -143,51 +175,55 @@ function BorderRadiusPanel({ selectedElements, pushUpdateForObject }) {
     : __('Top left corner radius', 'web-stories');
   return (
     <SimplePanel name="borderRadius" title={__('Corner radius', 'web-stories')}>
-      <BorderRow>
-        <BoxedNumeric
-          value={borderRadius.topLeft}
-          aria-label={firstInputLabel}
-          onChange={(value) => handleChange('topLeft', value)}
-        />
-        {!lockRadius && (
-          <>
-            <Icon>
-              <Union />
-            </Icon>
-            <BoxedNumeric
-              value={borderRadius.topRight}
-              aria-label={__('Top right corner radius', 'web-stories')}
-              onChange={(value) => handleChange('topRight', value)}
+      <FlexContainer>
+        <InputContainer>
+          <Row>
+            <BoxedNumericInput
```

Comment:
We should also use the new `isIndeterminate` attribute here for mixed multi-selection (when multiple elements are selected with different values) and use `MULTIPLE_DISPLAY_VALUE` as the `placeholder` in that case.

Reply:
You're totally right, thanks for catching that it was breaking in that case!

I ended up adding the functionality without the use of `isIndeterminate`. Lmk what you think!

Reply:
FI: Issue to address the 'mixed' value consistently across the panels and make changes to the `isIndeterminate` currently used (if needed): https://github.com/google/web-stories-wp/issues/6640